src/index.js

```diff
@@ -10,6 +10,7 @@ import stack from './lib/stack'
 import BarView from './components/Bar'
 
 const APP_SELECTOR = '[role=application]'
+const BANNER_SELECTOR = '[role=banner]'
```

Comment:
I'm not sure it's really useful as the `[role=banner]` element is the bar itself. When we need to rely on it, we assume the bar is present as we previously injected it, nope?